editor/rules/parser.ts

```diff
@@ -23,8 +21,8 @@ export class AaribaInterpreter {
     }
 
     execute(content: string) {
-        let globalsInit = clone(this.globals);
-        let locals = pegParser.parse(content, this.globals);
+        //let globalsInit = clone(this.globals);
```

Comment:
This is only my point of view, but rather than commenting a line of code, we can delete it: git is here for us and keep in mind what we might forget.